environment-setup.md

```diff
@@ -0,0 +1,152 @@
+# 💻 Setting up your development environment
+
+There are many guides to setting up your computer for development. Each has its own merits. At DataMade, we perform a mix of maintaining legacy projects and developing new projects, often in parallel. Our toolkit is therefore optimized for managing many, isolated versions of packages and getting up and running on new projects quickly.
+
+#### Contents
+
+- [Version control](#version-control)
+- [Text editor](#text-editor)
+- [Python](#python)
+- [Docker](#docker)
+- [Security](#security)
+- [Data](#data)
+- [Geospatial data](#geospatial-data)
+- [Static sites](#static-sites)
+
+For Linux users, most of these tools are available as aptitude packages. When possible, prefer binary packages to building from source. For Mac users, most of these tools are available as [Homebrew](http://brew.sh/) packages. Homebrew is by far the easiest way to manage packages on OS X.
+
+## Version control
+
+We use GitHub and Git to keep our work under version control. Note that we prefer the `git` CLI to the GitHub desktop GUI.
+
+#### Packages
+
+* [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+
+## Text editor
+
+Most of us use Sublime as our text editor of choice. If you do, too, then you will need to make some modifications to your settings.
```

Comment:
Nitpick, but I don't think this sentence is true anymore. More accurate might be:

```suggestion
DataMade developers use a variety of different text editors, and we leave it up to you to decide which one you prefer to use. However, there are two configurations that we encourage you to make no matter which editor you use:

 1. Automatically trim trailing whitespace
 2. Set default tab size for Python files to 4, and set default HTML/JavaScript tab size to 2
 
Historically, Sublime Text has been the most widely-used editor at DataMade, so the following instructions will demonstrate how to set these settings for Sublime Text.
```